stencil-library/storybook-pages/react.stories.mdx

```diff
@@ -0,0 +1,210 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import CSSVars from './css-variables.tsx';
+import { SummaryElement, extractVersionFromPackage } from './utils.tsx';
+
+<Meta title="React" />
+
+# Justifi React web components
+
+# Index
+
+- [Usage](#usage)
+- [Usage details](#usage-details)
+  - [Listening to events](#listening-to-events)
+  - [Calling methods](#calling-methods)
+- [Styling exported parts](#styling-exported-parts)
+
+# Usage
+
+To use the web-components in a React app you need to add the CDN script and the link to the default styles to your `index.html` file:
+
+<Source
+  language="bash"
+  dark
+  format={false}
+  code={dedent`
+    <head>
+      <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/@justifi/webcomponents@${extractVersionFromPackage()}/dist/webcomponents/webcomponents.css"
+      />
+      <script
+        async
+        type="module"
+        src="https://cdn.jsdelivr.net/npm/@justifi/webcomponents@${extractVersionFromPackage()}/dist/webcomponents/webcomponents.esm.js"
+      />
+    </head>
+  `}
+/>
+
+Then you can use the components as any other HTML element:
+
+<Source
+  language="jsx"
+  dark
+  format={false}
+  code={dedent`
+    <justifi-card-form
+      single-line="true"
+      validation-mode="onSubmit"
+    />
+`}
+/>
+
+# Usage details
+
+## Tyepscript integration
+
+To integrate with TypeScript, declare the component in the JSX.IntrinsicElements interface. This helps TypeScript understand the custom element, its attributes, and events.
+
+<Source
+  language="jsx"
+  dark
+  format={false}
+  code={dedent`
+    import React, { useState, DOMAttributes } from 'react';
+
+    type CustomElement<T> = Partial<T & DOMAttributes<T> & { children?: React.ReactNode }>;
+
+    declare global {
+      namespace JSX {
+        interface IntrinsicElements {
+          'justifi-payment-form': CustomElement<any>; // Specify more specific types as needed
+        }
+      }
+    }
+
+`}
+/>
+
+## Listening to events
```

Comment:
Should this section come after the "Calling Methods" section since it includes the instructions on creating a ref?

Reply:
Yes, it makes sense.